client/my-sites/domains/domain-management/dns/restore-default-a-records-dialog.tsx

```diff
@@ -1,8 +1,18 @@
 import { Dialog } from '@automattic/components';
-import { sprintf } from '@wordpress/i18n';
 import { useI18n } from '@wordpress/react-i18n';
+import type { ResponseDomain } from 'calypso/lib/domains/types';
```

Comment:
Converted this and the other dialog to TSX to support type checking

Reply:
Thanks for this 🙂